pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/WorkflowDao.sql.stg

```diff
@@ -1,15 +1,21 @@
 group WorkflowDao;
 
 queryOrderedActiveNextStateCandidatesByStudyIdAndFromStateId() ::= <<
-select trans.workflow_transition_id,
+select trans.workflow_transition_id, trans.from_state_id,
        next_state_type.workflow_state_type_code,
        next_act_state.study_activity_id,
+       next_redirect_state.study_guid,
+       next_redirect_state.redirect_url,
+       us_redirect.study_name,
```

Comment:
I think we might be better off getting it from the `StudyDto` instead. We're passing in `studyGuid` in `WorkflowService` but we should probably change that.